lib/assessment/rubric_tester.py

```diff
@@ -56,45 +80,47 @@ def get_passing_grades(num_passing_grades):
         return None
 
 
-def read_inputs(prompt_file, standard_rubric_file):
-    with open(prompt_file, 'r') as f:
+def read_inputs(prompt_file, standard_rubric_file, prefix):
+    with open(os.path.join(prefix, prompt_file), 'r') as f:
         prompt = f.read()
 
-    with open(standard_rubric_file, 'r') as f:
+    with open(os.path.join(prefix, standard_rubric_file), 'r') as f:
         standard_rubric = f.read()
 
     return prompt, standard_rubric
 
 
-def get_student_files(max_num_students, student_ids=None):
+def get_student_files(max_num_students, prefix, student_ids=None):
     if student_ids:
-        return [f"sample_code/{student_id}.js" for student_id in student_ids]
+        return [os.path.join(prefix, f"{student_id}.js") for student_id in student_ids]
     else:
-        return sorted(glob.glob('sample_code/*.js'))[:max_num_students]
+        return sorted(glob.glob(os.path.join(prefix, '*.js')))[:max_num_students]
 
 
-def get_expected_grades(expected_grades_file):
+def get_expected_grades(expected_grades_file, prefix):
     expected_grades = {}
-    with open(expected_grades_file, newline='') as csvfile:
+    with open(os.path.join(prefix, expected_grades_file), newline='') as csvfile:
         reader = csv.DictReader(csvfile)
         for row in reader:
             student_id = row['student']
             expected_grades[student_id] = dict(row)
     return expected_grades
 
 
-def get_examples():
-    example_js_files = sorted(glob.glob('examples/*.js'))
+def get_examples(prefix):
+    example_js_files = sorted(glob.glob(os.path.join(prefix, 'examples', '*.js')))
     examples = []
     for example_js_file in example_js_files:
         example_id = os.path.splitext(os.path.basename(example_js_file))[0]
         with open(example_js_file, 'r') as f:
             example_code = f.read()
-        with open(f"examples/{example_id}.tsv", 'r') as f:
+        with open(os.path.join(prefix, 'examples', f"{example_id}.tsv"), 'r') as f:
             example_rubric = f.read()
         examples.append((example_code, example_rubric))
     return examples
 
+# TODO: Add check if concepts in "examples" match rubric and expected concepts
+# Mark has done this- pull branch 
```

Comment:
Looks like this has been merged now. Maybe worth pulling in now to make sure there are no surprises in merge conflicts later?

Reply:
I already pulled Mark's changes in, so I'll just delete this comment